app/dialog/dialog.js

```diff
@@ -37,6 +41,33 @@ export default class Dialog extends crsbinding.classes.ViewBase {
         });
     }
 
+    async showNestedDialog(event) {
```

Comment:
Double check we're implementing the nested scenario properly here as per discussed

Reply:
checked the scenarios and implemented a few changes to resolve issues encountered.